packages/server/modules/workspacesCore/domain/types.ts

```diff
@@ -12,4 +12,10 @@ export type Workspace = {
 
 export type WorkspaceWithOptionalRole = Workspace & { role?: WorkspaceRoles }
 
-export type WorkspaceAcl = { userId: string; role: WorkspaceRoles; workspaceId: string }
+export type WorkspaceAcl = {
+  userId: string
+  role: WorkspaceRoles
+  workspaceId: string
+  createdAt: Date
+  updatedAt: Date
```

Comment:
as discussed, we only need createdAt, updatedAt has not usefull meaning for now.

Reply:
created the `workspace_acl_updates` table as an exercise - now we can keep `workspace_acl` unchanged